src/components/navigation/Precedent.tsx

```diff
@@ -0,0 +1,26 @@
+import Button from '@codegouvfr/react-dsfr/Button';
+import { OrchestratedElement } from '../orchestrator';
+
+export function Precedent(props: OrchestratedElement) {
+	const { goPreviousPage = () => null, isFirstPage } = props;
+
+	function handleClick() {
+		goPreviousPage();
+	}
+	if (!isFirstPage) {
+		return (
+			<div className="fr-grid-row fr-grid-row--center fr-grid-row--middle fr-mt-md-2w ">
```

Comment:
Maybe the divs should be on the wrapper component ?

Reply:
les règles de rang de la grille ne peuvent pas être les mêmes sur tout les éléments de la page.
On ne pourra pas mutualiser sur le composant parent, désolé.